test/strategies/simple-test.js

```diff
@@ -15,23 +43,28 @@ describe('SimpleConcat', function() {
 
   it('can handle empty input scenarios', function() {
     let concat = new SimpleConcat({});
-    concat.addFile('foo.js', '');
+    let fileContent = '';
+    let obj = {
+      'a.js': fileContent
+    };
+    writeFixturifySync(obj);
+    concat.addFile(`${testDir}/a.js`, fileContent);
```

Comment:
instead of `${testDir}` in all this `addFiles` could we specify `baseDir` when creating `SimpleConcat`?

Reply:
Updated. I feel stupid for not doing something so obvious